### PR TITLE
Add translators.json

### PR DESCRIPTION
**Pull request by skrlet13**
I made a .json file with translators and their links (if they had). They are grouped by language.

### CODE REVIEW

File: translators.json [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+        {"name": "Néfix Estrada", "link": ""},
+	{"name": "spla", "link": "https://weblate.yerbamate.ml/user/spla/"}
+    ],
+	"da": [
+		{
+		}
+	],
+	
+	"de": [
+		{"name":"0xflotus", "link":""},
```
Comment (dessalines) [inline translators.json:16]: Could you remove these empty link fields?

**Review by dessalines — CHANGES_REQUESTED** [review]

File: translators.json (hunk 1 above)
Comment (skrlet13) [reply translators.json:16]: Ok, so i leave this like this? :
` {"name":"0xflotus"},`

Comment (dessalines) [reply translators.json]: Ya

File: translators.json [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+	{"name": "spla", "link": "https://weblate.yerbamate.ml/user/spla/"}
+    ],
+	"da": [
+		{
+		}
+	],
+	
+	"de": [
+		{"name":"0xflotus"},
+		{"name":"T", "link":""}
```
Comment (dessalines) [inline translators.json:17]: This one still

**Review by dessalines — CHANGES_REQUESTED** [review]

**Comment by dessalines** [comment]
Sweet thx! I did some more formatting and removing empty keys, but everything looks good.

**Review by dessalines — APPROVED** [review]

**Pull request merged**